### PR TITLE
Google Drive to local - system tests migrations (AIP-47)

### PR DESCRIPTION
**Pull request by bkossakowska**
Google Drive to local - system tests migrations (AIP-47)

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+
+FILE_NAME = "empty.txt"
+OUTPUT_FILE = "out_file.txt"
+DAG_ID = "example_gdrive_to_local"
+BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
+
+CURRENT_FOLDER = Path(__file__).parent
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py:41]: Do you use this variable somewhere?

File: tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+LOCAL_PATH = str(Path(__file__).parent / "resources" / FILE_NAME)
+
+with models.DAG(
+    DAG_ID,
+    start_date=datetime(2021, 1, 1),
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py:46]: schedule=@once missing, without schedule it will not run

**Comment by bhirsz** [comment]
Please also remove old test_<service_name>_system.py file

File: tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py (hunk 1 above)
Comment (bkossakowska) [reply tests/system/providers/google/cloud/transfers/example_gdrive_to_local.py:41]: removed

**Comment by bkossakowska** [comment]
> Please also remove old test_<service_name>_system.py file

no file: test_<service_name>_system.py

**Pull request closed without merging**